jupyter_scheduler/extension.py

```diff
@@ -60,6 +59,13 @@ def _db_url_default(self):
         help=_i18n("The scheduler class to use."),
     )
 
+    job_files_manager_class = Type(
+        default_value="jupyter_scheduler.job_files_manager.JobFilesManager",
+        klass="jupyter_scheduler.job_files_manager.JobFilesManager",
```

Comment:
Is 'klass' spelling a typo?

Reply:
Yes, but I think it's deliberate, to avoid conflict with the reserved word `class` in Python